src/trio/_tests/test_socket.py

```diff
@@ -324,7 +324,9 @@ async def test_sniff_sockopts() -> None:
     sockets = []
     for family in [AF_INET, AF_INET6]:
         for type_ in [SOCK_DGRAM, SOCK_STREAM]:
-            sockets.append(stdlib_socket.socket(family, type_))
+            sockets.append(  # noqa: PERF401  # Makes sense here, clarity.
+                stdlib_socket.socket(family, type_)
+            )
```

Comment:
We could do:
```py
sockets = [
    stdlib_socket.socket(family, type_)
    for family in [AF_INET, AF_INET6]
    for type_ in [SOCK_DGRAM, SOCK_STREAM]
]
```
Or actually, since it's just immediately used, just change this to use `itertools.product()`, make the stdlib socket call the first line.

Reply:
Thanks, added in 550518a